repository/p.json

```diff
@@ -643,6 +643,16 @@
 				}
 			]
 		},
+		{
+			"name": "PDML",
+			"details": "https://github.com/tajmone/Sublime-PDML",
```

Comment:
You'd want a label there to indicate the kind of package.

Reply:
Done! (also rebased on latest master head before amending and force pushing)